Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -1024,6 +1026,34 @@ void EditorActionsHandler::OnActionRegistrationHook()
         m_hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "Ctrl+Space");
     }
 
+    // Only Show Helpers for Selected Entities
+    {
+        const AZStd::string_view& actionIdentifier = "o3de.action.view.toggleSelectedEntityHelpers";
+
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "Show Helpers for Selected Entities Only";
+        actionProperties.m_description = "Show Helpers for Selected/All Entities";
```

Comment:
Suggestion for the following reasons:
- Better informs user of which toggle sets to which setting. 
- Don't need to capitalize the description in the same way as the name. Since this is a normal sentence, capitalization usually only applies to the first word and proper names. 
- I know most tooltip descriptions don't have a period at the end -- not sure if this is standard style for code references? If so, you can remove the period in the last sentence. 
```suggestion
        actionProperties.m_description = "If enabled, shows Helpers for selected entities only. By default, shows Helpers for all entities.";
```